scripts/zones/AlTaieu/mobs/Aw_euvhi.lua

```diff
@@ -5,8 +5,9 @@
 package.loaded["scripts/zones/AlTaieu/TextIDs"] = nil;
 -----------------------------------
 
-require("scripts/zones/AlTaieu/TextIDs");
 require("scripts/globals/keyitems");
+require("scripts/zones/AlTaieu/MobIDs");
+require("scripts/zones/AlTaieu/TextIDs");
```

Comment:
*sniffle*

reversing my upside down pyramids of headers again

😢

Reply:
Early birthday present.

Reply:
didnt hafta but I lurva ya for it<3